pkg/core/function.go

```diff
@@ -43,28 +43,31 @@ import (
 )
 
 const (
-	functionLabel   = "riff.projectriff.io/function"
-	buildAnnotation = "riff.projectriff.io/nonce"
-	// annotation names with slashes are rejected :-/
+	functionLabel                 = "riff.projectriff.io/function"
+	buildAnnotation               = "riff.projectriff.io/nonce"
 	buildpackBuildImageAnnotation = "riff.projectriff.io-buildpack-buildImage"
 	buildpackRunImageAnnotation   = "riff.projectriff.io-buildpack-runImage"
+	functionArtifactAnnotation    = "riff.projectriff.io/artifact"
+	functionOverrideAnnotation    = "riff.projectriff.io/override"
```

Comment:
Should this be modified for consistency with the other build option annotations? E.g.
```
functionInvokerAnnotation = "riff.projectriff.io/invoker"
```

Reply:
There is some discrepancy between riff.toml and the CLI terms. The name "invoker" became wrong imo, override is better. So we need to decide at which point we switch from one to the other.